.ci/schedule-weekly.groovy

```diff
@@ -75,25 +82,37 @@ pipeline {
   }
 }
 
+def runNotifyStalledBeatsBumps(Map args = [:]){
+  def branch = getMajorMinorGivenTheBranch(args)
+  notifyStalledBeatsBumps(branch: branch,
+                          sendEmail: false,
+                          to: 'beats-contrib@elastic.co')
```

Comment:
For the time being, I'll enable this only in my email so I can verify everything works as expected
```suggestion
                          sendEmail: true,
                          to: 'victor.martinez@elastic.co')
```